wlct/management/commands/engine.py

```diff
@@ -72,6 +74,94 @@ def parse_and_update_clan_logo():
     except Exception:
         log_exception()
 
+def is_correct_player(player_token, player_team):
+    try:
+        player = Player.objects.filter(token=player_token)[0]
+        tt = TournamentTeam.objects.filter(pk=player_team)[0]
+        if player.clan and player.clan.name == tt.clan_league_clan.clan.name:
+            return True
+
+        tplayers = TournamentPlayer.objects.filter(team=tt)
+        for tplayer in tplayers:
+            if tplayer.player.token == player_token:
+                return True
+        return False
+    except:
+        log_exception()
+        return False
+
+
+def is_valid_team_order(team_order, team_dict, teams):
+    for i in range(len(team_order)):
+        for player in team_dict[team_order[i]]:
+            if is_correct_player(player, teams[i]):
+                return True
+    return False
+
+def validate_cl_tournament_games():
+    try:
+        print("Running validate_cl_tournament_games on thread {}".format(threading.get_ident()))
+        tournament_games = TournamentGame.objects.filter(is_finished=False)
+        api = API()
+        found_missing_games = 0
+        corrected_missing_games = 0
+        test_content = TestContent()
+
+        # Iterate through unfinished games and check of missing player values
+        for game in tournament_games:
+            teams = game.teams.split('.')
+            tt = TournamentTeam.objects.filter(pk=teams[0])
+            if not tt or tt[0].clan_league_clan:
```

Comment:
can remove the .clan_league_clan check since we know they have to have them if you alter the filter above.